Description: disable tests related to deprecation
 This feature is disable
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-07-04

--- a/test/release.test.js
+++ b/test/release.test.js
@@ -107,21 +107,6 @@
   warn(warn.called, false)
 })
 
-test('shows warning', () => {
-  browserslist.data = olderSixMonthsData
-  spyOn(fs, 'existsSync', findPackage)
-  spyOn(fs, 'statSync', mockStatSync)
-  browserslist('last 2 versions')
-  equal(warn.calls, [
-    [
-      'Browserslist: caniuse-lite is outdated. Please run:\n' +
-        '  npx update-browserslist-db@latest\n' +
-        '  Why you should do it regularly: ' +
-        'https://github.com/browserslist/update-db#readme'
-    ]
-  ])
-})
-
 test('hides warning on request', () => {
   process.env.BROWSERSLIST_IGNORE_OLD_DATA = 'true'
   browserslist.data = olderSixMonthsData
@@ -131,13 +116,4 @@
   is(warn.called, false)
 })
 
-test('shows warning only once', () => {
-  browserslist.data = olderSixMonthsData
-  spyOn(fs, 'existsSync', findPackage)
-  spyOn(fs, 'statSync', mockStatSync)
-  browserslist('last 2 versions')
-  browserslist('last 2 versions')
-  is(warn.callCount, 1)
-})
-
 test.run()
